code-challenges.test.js

```diff
@@ -35,9 +72,41 @@ const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+describe("oddElevation", () => {
+
+    it(" oddElevation will take arrays fullArr2, oddElevation will return a new array containing only odd numbers sorted least to greatest", () => {
+
+        expect(oddElevation()).toEqual([-9, 7, 9, 199])
+
+    })
+})
+
```

Comment:
Your test is good it's just missing the argument in the oddElevation() function.  Add that and you're good to go!